### ISSUE TITLE
[SDL2] Saitek X55 makes OpenTK to crash

### ISSUE THREAD

**Issue by mrhelmut** [opentk/opentk#377]
Hello,

When a Saitek X55 is plugged, it looks like it is recognized as a gamepad and crashes when trying to get its state.

```
   at OpenTK.Input.JoystickState.SetButton(JoystickButton button, Boolean value)
   at OpenTK.Platform.SDL2.Sdl2JoystickDriver.OpenTK.Input.IJoystickDriver2.GetState(Int32 index)
   at OpenTK.Platform.MappedGamePadDriver.GetState(Int32 index)
```

Haven't tested this on other backends.

**Comment by PERECil** [comment]
I can confirm that the same issue arise for Saitek X52.

**Comment by leezer3** [comment]
It's not actually the GamePad driver at fault-
https://github.com/opentk/opentk/pull/344

TLDR:
There's a 32 button, 8 axis hard-limit in OpenTK. 
The pull above ups this to 128 buttons and 32 axis.
There's some irrelevant junk at the bottom where I ran into some behaviour I wasn't expecting with the Linux joystick driver, but the pull above fixes the issue.

**Comment by varon** [comment]
Probably fixed by #453.
